docs/pages/reference/backends.mdx

```diff
@@ -620,6 +620,282 @@ Config option priority is applied in the following order:
+      - 'stdout://'
+```
+
+```
+athena://db.table?topicArn=arn:aws:sns:region:account_id:topic_name&largeEventsS3=s3://transient/large_payloads&locationS3=s3://long-term/events&workgroup=workgroup&queueURL=https://sqs.region.amazonaws.com/account_id/queue_name&queryResultsS3=s3://transient/query_results
+```
+
+The URL hostname consist of `database.table`, which points to the Glue database
+and a table which will be used by the Athena audit logger.
+
+Other parameters are specified as query parameters to Athena url.
+
+The following parameters are required:
+- `topicArn=arn:aws:sns:region:account_id:topic_name` - ARN of SNS topic where events are published
+- `locationS3=s3://long-term/events` - S3 bucket used for long term storage
+- `largeEventsS3=s3://transient/large_payloads` - S3 bucket used for transient storage for large events
+- `queueURL=https://sqs.region.amazonaws.com/account_id/queue_name` - SQS URL used as subscription to SNS topic
+- `workgroup=workgroup_name` - Athena workgroup used for queries
+- `queryResultsS3=s3://transient/results` - S3 bucket used for transient storage for query results
+
+Following parameters are optional:
+- `region=us-east-1` - AWS region, if empty defaults to one from AuditConfig or AWS Credentials
+- `batchMaxItems=20000` - defines maximum number of events allowed for single parquet file (default 20000).
+- `batchMaxInterval=1m` - defines maximum interval used to buffer incoming data before creating Parquet file (default 1m).
+
+### Authenticating to AWS
+
+The Teleport Auth Service must be able to read AWS credentials in order to
+authenticate to Athena.
+
+(!docs/pages/includes/aws-credentials.mdx service="the Teleport Auth Service"!)
+
+The IAM role that the Teleport Auth Service authenticates as must have the
+policies specified in the next section.
+
+### IAM policies
+
+Make sure that the IAM role assigned to Teleport is configured with sufficient
+access to Athena. Below you can find the IAM permissions that the Auth Service
+requires to use Athena Audit logs as an audit event backend.
+
+You'll need to replace these values in the policy example below:
+
+| Placeholder value | Replace with                                             |
+|-------------------|----------------------------------------------------------|
+| `eu-central-1`    | AWS region                                               |
+| `1234567890`      | AWS account ID                                           |
+| `audit-long-term` | S3 bucket used for long term storage                     |
+| `audit-transient` | S3 bucket used for transient storage                     |
+| `audit-sqs`       | SNS topic name                                           |
+| `audit-sns`       | SQS name                                                 |
+| `kms_id`          | KMS key ID used for server side encryption of SNS/SQS/S3 |
+| `audit_db`        | Glue database name user for audit logs                   |
```

Comment:
Should "user" be "used" here? "name user" seems odd to me

Reply:
good catch, come copy paste error, reworked to:

> Glue database used for audit logs

same for table